ding/worker/learner/learner_hook.py

```diff
@@ -202,10 +202,11 @@ def __call__(self, engine: 'BaseLearner') -> None:  # noqa
             - engine (:obj:`BaseLearner`): the BaseLearner
         """
         # Only show log for rank 0 learner
-        if engine.rank != 0:
-            for k in engine.log_buffer:
-                engine.log_buffer[k].clear()
-            return
+        # if engine.rank != 0:
```

Comment:
also pass the `only_monitor_rank0` argument to the hook class

Reply:
done